map/style_tests/style_symbols_consistency_test.cpp

```diff
@@ -14,37 +14,38 @@
 #include "coding/parse_xml.hpp"
 #include "coding/reader.hpp"
 
+#include <cstring>  // std::strcmp
 #include <set>
 #include <string>
 #include <vector>
 
-using namespace std;
-
 namespace style_symbols_consistency_tests
 {
+typedef std::set<std::string> StringSet;
+
 class SdfParsingDispatcher
 {
 public:
-  explicit SdfParsingDispatcher(set<string> & symbols)
+  explicit SdfParsingDispatcher(StringSet & symbols)
       : m_symbols(symbols)
   {}
 
-  bool Push(string const &) { return true; }
-  void Pop(string const &) {}
-  void CharData(string const &) {}
-  void AddAttr(string const & attribute, string const & value)
+  bool Push(char const *) { return true; }
+  void Pop(char const *) {}
+  void CharData(std::string const &) {}
+  void AddAttr(char const * attribute, char const * value)
```

Comment:
```std::string_view attribute``` for nicer compare?

Reply:
The current approach is marginally faster :)